source/BoardingPanel.h

```diff
@@ -98,7 +100,7 @@ class BoardingPanel : public Panel {
 		void Take(int count);
 
 	private:
-		void UpdateStrings();
+		void UpdateStrings(bool obscureValue);
```

Comment:
```suggestion
		void UpdateStrings(bool obscureValue = false);
```
if you make the value default to false you dont have to specify its false at some places